metis/bin/metis_client

```diff
@@ -1076,6 +1079,15 @@ EOT
 
   class Put < Command
     usage "put <input_file1> ... [<input_fileN>] <remote_folder_path>"
+
```

Comment:
Normally metis and the client will barf on badly-named files; this attempts to resolve by replacing illegal characters with `_`.